src/configuration.rs

```diff
@@ -55,10 +55,10 @@ impl Default for Configuration {
                 minimal_version: String::new(),
             },
             file_lister_config: FileListerConfig {
-                dir: vec![PathBuf::from("src")],
+                dir: vec![PathBuf::from("tests/assets/test_folder")],
```

Comment:
I suggest to not modify it, the configuration file has been implemented for this reason

Reply:
As discussed on discord, since we don't ship the code with the executable it's better to put the test folder